extensions/mssql/package.nls.json

```diff
@@ -179,8 +179,14 @@
 	"title.designTable": "Design",
 	"mssql.parallelMessageProcessing" : "[Experimental] Whether the requests to the SQL Tools Service should be handled in parallel. This is introduced to discover the issues there might be when handling all requests in parallel. The default value is false. Relaunch of ADS is required when the value is changed.",
 	"mssql.tableDesigner.preloadDatabaseModel": "Whether to preload the database model when the database node in the object explorer is expanded. When enabled, the loading time of table designer can be reduced. Note: You might see higher than normal memory usage if you need to expand a lot of database nodes.",
-
 	"mssql.objectExplorer.groupBySchema": "When enabled, the database objects in Object Explorer will be categorized by schema.",
 	"mssql.objectExplorer.enableGroupBySchema":"Enable Group By Schema",
-	"mssql.objectExplorer.disableGroupBySchema":"Disable Group By Schema"
+	"mssql.objectExplorer.disableGroupBySchema":"Disable Group By Schema",
+	"title.newServerRole": "New Server Role",
+	"title.newLogin": "New Login",
```

Comment:
can we add the string "Preview" to these commands for now?

Reply:
I've already added Preview to the dialog title.
![image](https://user-images.githubusercontent.com/13777222/219798821-a10f32b1-9f78-4da0-8bd1-46a57e942e14.png)